packages/sdks/src/components/content-variants/content-variants.lite.tsx

```diff
@@ -92,6 +96,13 @@ export default function ContentVariants(props: VariantsProviderProps) {
           id="builderio-init-variants-fns"
           nonce={props.nonce || ''}
         />
+        {SDKS_SUPPORTING_PERSONALIZATION.includes(TARGET) && (
+          <InlinedScript
+            nonce={props.nonce || ''}
+            scriptStr={getInitPersonalizationVariantsFnsScriptString()}
+            id="builderio-init-personalization-variants-fns"
```

Comment:
existing issue in a/b tests potentially as well: 

these scripts set stuff on `window.*`, which makes me think things might break if users render:

```tsx
<Content /> 
<Content />
```

multiple `Content` components in the same page.

Reply:
@samijaber just checked, we don't use ids of the scripts anywhere in the inlined scripts. These global window fn initialisations will be defined twice if we use two Contents and it will override the function again in window but not affect the contents. 

https://github.com/sidmohanty11/builder/blob/general-support-vc/packages/sdks/src/components/content-variants/inlined-fns.ts/#L142 - we attach cookies to a specific content id and content ids will be different in each case if multiple Content are using A/B tests feature